### PR TITLE
Update readme: getting started

### PR DESCRIPTION
**Pull request by richardkapolnai-da**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,9 +43,9 @@ Telegram is used as an optional receive confirmation messages from the app. Skip
     3. Create your own bot by sending the `/newbot` message to the `BotFather`. Give it a unique name ending in bot. The new bot will appear in your Telegram contact list using this name.
 
 5.  Note the token that is displayed. This will be your bot token. The usual format is:
-```
+    ```
```
Comment (tamaskalcza-da) [inline README.md]: Maybe one ` is enough.

File: README.md [hunk 2]
```diff
@@ -55,104 +55,95 @@ Telegram is used as an optional receive confirmation messages from the app. Skip
 
 8. Specify the details of your Telegram setup.
 
-```
-cp -n telegram.properties.sample telegram.properties
-```
-Edit this file in a plain text editor and add the values for
-`BOT_TOKEN` and `CHAT_ID` you acquired in step 1 above.
+   ```
```
Comment (tamaskalcza-da) [inline README.md]: For consistency:
```suggestion
   ```shell
```

File: README.md [hunk 3]
```diff
@@ -55,104 +55,95 @@ Telegram is used as an optional receive confirmation messages from the app. Skip
...
+1. Stop the Docker containers or bots by pressing **Ctrl+C**. (Alternatively, you can also stop it by typing `docker-compose down`.)
 
-
-#### Stopping Stand-Alone Run
-1. Close the browser tab.
-2. Stop the bots by pressing **Ctrl+C**.
-3. Stop the Sandbox and the Navigator by pressing **Ctrl+C**.
+#### Stopping Standalone Run
+1. Stop the bots by pressing **Ctrl+C**.
+1. Stop the Sandbox and the Navigator by pressing **Ctrl+C** in the DAML assistant.
```
Comment (tamaskalcza-da) [inline README.md]: Numbering inconsistency.

File: README.md (hunk 2 above)
Comment (richardkapolnai-da) [reply README.md]: thanks

File: README.md (hunk 3 above)
Comment (richardkapolnai-da) [reply README.md]: for easier copy paste, I cheated

File: README.md [hunk 4]
```diff
@@ -55,104 +55,95 @@ Telegram is used as an optional receive confirmation messages from the app. Skip
...
+   `cd ./output_messages
+   `ls -l
+   ````
 5. Print file content.
-```
-cat <file_ name>
-```
+   ````
+   `cat <file_ name>
+   ````
```
[the remark this answers is not in the document]
Comment (richardkapolnai-da) [reply README.md]: good catch, on it

File: README.md (hunk 1 above)
Comment (richardkapolnai-da) [reply README.md]: maybe, I wanted to do as small change as possible

**Pull request merged**